cmd/otelcol/config/collector/gateway_config.yaml

```diff
@@ -142,6 +142,22 @@ exporters:
     token: "${SPLUNK_ACCESS_TOKEN}"
     endpoint: "${SPLUNK_INGEST_URL}/v1/log"
     log_data_enabled: false
+  # To send entities (applicable only if discovery mode is enabled)
+  otlphttp/entities:
+    logs_endpoint: "${SPLUNK_INGEST_URL}/v3/event"
+    headers:
+      "X-SF-Token": "${SPLUNK_ACCESS_TOKEN}"
+    auth:
+      authenticator: headers_setter
+
+connectors:
+  # Routing connector to separate entity events from regular logs
+  routing/logs:
+    default_pipelines: [logs]
+    table:
+      - context: log
+        condition: instrumentation_scope.attributes["otel.entity.event_as_log"] == true
```

Comment:
Where does `otel.entity.event_as_log` come from? Is there some way we could use a central variable or constant here? Not required as it might be a bit messy, just concerned about the possibility of this changing in the future.

Reply:
This is coming from the sender (agent where discovery is enabled).
We can use env variable, but I don't think it's worth it